src/pages/builders/challenges.tsx

```diff
@@ -1,17 +1,81 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import type { GetServerSideProps, NextPage } from 'next'
+import moment from 'moment'
+import Phases from '@/data/phase'
+import { Accordion } from '@/components/index'
+import { useAccordion } from '@/hook/useAccordion'
+import { ContentBlock, taskContentIcon } from '../validators/tasks'
```

Comment:
The `challenges` page should never import some elements from another page, please create `ContentBlock` and `taskContentIcon` components into a dedicated folder into the view itself as an example.